### ISSUE TITLE
Support for parameters

### ISSUE THREAD

**Issue by dhoer** [chocolatey/chocolatey-test-environment#26]
While it is desirable to not require parameters, sometimes it is necessary.  I have a package that requires a password and there is no way to pass that information to test environment.  

I would like to propose support for a properties file e.g., `tools\choco_test.properties` that is optional and would be converted to `--params "''"` automatically before executing the command.

**Comment by dhoer** [comment]
I'm not sure why adding prompt for password would make the test pass, but it did: https://gist.github.com/83b5b44e6c2e9520852112a2d72e33c5.

**Comment by ferventcoder** [comment]
Agreed. That is weird, unless in doing so you may have snuck an emtpy default password in.
